cypress/integration/treasuries_spec.ts

```diff
@@ -86,10 +87,36 @@ describe('Create a new treasury', () => {
     cy.contains(editedDesc).should('be.visible')
   })
 
+  it('Should export user data', () => {
+    cy.visit('http://localhost:3000/config')
+    cy.contains('Export User Data').click()
+    cy.wait(2000)
+    const downloadsFolder = Cypress.config('downloadsFolder')
+    const downloadedFilename = path.join(downloadsFolder, 'roundtable-backup.mainnet.json')
+    cy.readFile(downloadedFilename).then(data => {
```

Comment:
You could delete this because the JSON content doesn't have to be tested in this black box testing.